addons/hr_timesheet/report/report_timesheet_templates.xml

```diff
@@ -106,7 +106,7 @@
         <t t-foreach="subtasks.sorted('sequence')" t-as="subtask">
             <t t-if="subtask in timesheets_per_task">
                 <h2 class="my-4">
-                    Sub-Task: <t t-out="subtask.name"/>
+                    Sub-Task of '<t t-out="title"/>': <t t-out="subtask.name"/>
```

Comment:
Just to be sure, Alexandra asked to add that in your fix. 🤔 

(because this change sounds like a small improvement 😄 )

Reply:
@xavierbol No she didn't, I was originally only supposed to fix 18.0 where the report is just blank. The reason it was blank is that someone added a condition not to display tasks with no recorded hours (Which also accidentally  removed the subtasks).

I just figured that since in 17.2 and 17.4 the parent task used to show up even with no hours it was a probably a bug as well, and since that means the parent task isn't always displayed I wanted to keep a trace of the hierarchy that could be lost in the process.

I uh, took some creative liberties here, it wasn't explicitly asked for

Reply:
Okay, I see. 🙂 

Thanks! 🙂